components/publish/PublishWidget.vue

```diff
@@ -97,6 +97,8 @@ async function toggleSensitive() {
 }
 
 async function publish() {
+  if (isPublishDisabled)
```

Comment:
can you move this check to `publishDraft` on `publish.ts` module'? rn only used here...

Reply:
> can you move this check to `publishDraft` on `publish.ts` module? rn only used here...

sure, I have resolved it.